.env.dev.template

```
@@ -4,11 +4,20 @@
 # This file contains overrides for local development (running Go outside Docker)
 # Usage: make server (automatically loads .env then .env.dev)
 
-# Database - use localhost instead of Docker service names
+# Database - use localhost instead of Docker service names, and the host-side
+# port mappings from docker-compose.yml instead of the internal 5432.
 DB_CORE_HOST=localhost
 DB_HOST=localhost
 DB_HOST_CERTIFICATE=localhost
 DB_PORT_CERTIFICATE=5439
+DB_INTERVIEW_HOST=localhost
+DB_HOST_INTERVIEW=localhost
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_HOST_INTERVIEW key should go before the DB_INTERVIEW_HOST key

(UnorderedKey)